### ISSUE TITLE
HW Huffman: suspicious README

### ISSUE THREAD

**Issue by vsalavatov** [evlinsky/cpp#13]
@LDVSOFT 
Задание требует хранить тестирующие исходники в папке test/ и создать исполняемый файл test, однако, кажется, запрещено иметь папку и файл с одинаковым именем.

**Comment by LDVSOFT** [comment]
Agreed, use any other name that makes sense. The task will be updated.
